app/web-components/src/client/preview/types.ts

```diff
@@ -19,3 +21,18 @@ export interface ShowErrorArgs {
   title: string;
   description: string;
 }
+
+/**
```

Comment:
Could this be added to a types-6-0.ts file instead ? This might change in the 6.1 release